packages/playback-core/src/index.ts

```diff
@@ -626,29 +658,41 @@ export const getDRMConfig = (
           ...supportedConfigurations,
         ];
       }
-      return navigator.requestMediaKeySystemAccess(keySystem, supportedConfigurations);
+      return navigator.requestMediaKeySystemAccess(keySystem, supportedConfigurations).then((value) => {
+        const drmType = toDRMTypeFromKeySystem(keySystem);
+        drmTypeCb?.(drmType);
+        return value;
+      });
     },
   };
 };
 
 export const getAppCertificate = async (appCertificateUrl: string) => {
   const resp = await fetch(appCertificateUrl);
+  if (resp.status !== 200) {
+    return Promise.reject(resp);
```

Comment:
NOTE: we need some way to error with enough info to introspect why the request failed. Currently simply rejecting with the non-200 `Response`, but we can definitely consider alternatives. Since this function scope doesn't have sufficient information to fully validate information (e.g. a mismatched `playbackId` + JWT `sub`), we can't do a full translation here without passing in additional context.